addons/jam/CfgWeapons.hpp

```diff
@@ -71,6 +82,10 @@ class CfgWeapons {
         magazineWell[] = {"CBA_65x39_Mk200"};
     };
 
+    class LMG_Zafir_F : Rifle_Long_Base_F {
+        magazineWell[] = {"CBA_762x51_LINKS", "CBA_762x54R_LINKS"}
```

Comment:
This is wrong. It's either 7.62x51NATO or 7.62x54R.

Reply:
In reality, the Negev NG7 uses 7.62x51 NATO, in the game Zafir uses 7.62x54R.

I think it should support both, so that mission designers can decide if they want to give players NATO or Russian ammo, depending on the scenario.

Reply:
So in game it's a modified Negev NG7 modified to use 7.62x54R. Why not just leave it at that ? More plausible than the weapon being able to use 2 very different ammo types.

Reply:
Because it isn't:

![grafik](https://user-images.githubusercontent.com/29610495/48001187-18641400-e108-11e8-8051-0fb870430c34.png)

Reply:
Yet the vanilla config would disagree:
`    class LMG_Zafir_F : Rifle_Long_Base_F {
        magazines[] = {"150Rnd_762x54_Box", "150Rnd_762x54_Box_Tracer"};
        descriptionShort = "Light Machine Gun<br />Caliber: 7.62x54 mm";
};
`

Reply:
In game ui text and usage trumps whatever is written on the texture. And being able to use both is super weird.

Reply:
I removed it, maybe I'll just create a copy of it with x54 in a mod of my own.